Source/ACE/Entity/Creature.cs

```diff
@@ -4,22 +4,25 @@
 using ACE.Managers;
 using ACE.Network;
 using ACE.Network.Enum;
-using ACE.Network.GameAction;
 using ACE.Network.GameEvent.Events;
 using ACE.Network.Motion;
 using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
 
 namespace ACE.Entity
 {
     public class Creature : Container
     {
-        public CreatureAbility Strength {
+        public CreatureAbility Strength
+        {
             get { return AceObject.StrengthAbility; }
             set { AceObject.StrengthAbility = value; }
         }
 
+        public AceObject AceCopse
```

Comment:
I really don't like exposing the AceObject as public.  Put a TODO to refactor, and I'll be okay with it.